app/models/user.rb

```diff
@@ -1,8 +1,13 @@
 class User < ApplicationRecord
   # has_many :posts, class_name: 'post', foreign_key: 'author_id'
-  has_many :comments, foreign_key: :author_id
-  has_many :likes, foreign_key: :author_id
-  has_many :posts, foreign_key: :author_id
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly make sure that `:confirmable` is enabled in the user model, based on the requirements a user should be asked for confirmation of email.